.codeclimate.yml

```
@@ -0,0 +1,13 @@
+version: "2"         # required to adjust maintainability checks
+exclude_patterns:
+ - "**/node_modules/"
+ - "static/js/ckeditor/"
+ - "db/"
+ - "**/test/"
+checks:
+  similar-code:
+    config:
+      threshold: 160
```

This one too, We might want to lower it later, but for now let's set this a bit higher.
```suggestion
      threshold: 200
```